src/DynamoPackages/PackageLoader.cs

```diff
@@ -211,6 +211,19 @@ private void ScanPackageDirectories(string root, IPreferences preferences)
             {
                 if (!Directory.Exists(root))
                 {
+                    string extension = null;
+                    if (root != null)
```

Comment:
if root is a string, then there should be another check for "" or string.empty

Reply:
In this case the call to Path.GetExtension takes care of checking for "" or string.empty. The check for null is just to avoid Path.GetExtension throwing an exception due to root being null.